modules/virt-installing-a-watchdog.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-a-watchdog.adoc
+
+[id="virt-defining-a-watchdog_{context}"]
+= Installing a watchdog
+
+To activate a watchdog card, install the watchdog package on your virtual machine and start the watchdog service.
```

Comment:
```suggestion
To activate a watchdog device, install the `watchdog` package on your virtual machine and start the watchdog service.
```

Let us use a constant nomenclature, and call this a watchdog device rather than watchdog card (unless you think "card" is better, I mostly care about consistency)

Reply:
Device works for me. I'll update across the doc.